cmd/fs-v1-metadata.go

```diff
@@ -315,37 +317,63 @@ func createFormatFS(fsPath string) error {
 }
 
 func initFormatFS(fsPath string) (rlk *lock.RLockedFile, err error) {
+	// Done channel is used to close any lingering retry routine, as soon
```

Comment:
the unit test is a separate program holds a lock.  It has no difference whether its a separate process or not.  Unable to understand `no meaning` part.